### ISSUE TITLE
Assertion failed '!foundDiff' in 'OpenSsl:GetSslError(Microsoft.Win32.SafeHandles.SafeSslHandle,int,byref):int' during 'Linear scan register alloc'

### ISSUE THREAD

**Issue by VSadov** [dotnet/runtime#39484]
In   Libraries Test Run checked coreclr Linux_musl arm64 Release

Failed here:
https://helix.dot.net/api/2019-06-17/jobs/844320ef-6ca1-4f63-8a1d-ea6c61cde3c9/workitems/System.Net.Requests.Tests/console
https://helix.dot.net/api/2019-06-17/jobs/844320ef-6ca1-4f63-8a1d-ea6c61cde3c9/workitems/System.Net.Security.Tests/console

Also here: 
https://helix.dot.net/api/2019-06-17/jobs/81d8492e-f2a0-461c-8f6e-5f4060c57b55/workitems/System.Net.Requests.Tests/console
https://helix.dot.net/api/2019-06-17/jobs/81d8492e-f2a0-461c-8f6e-5f4060c57b55/workitems/System.Net.Security.Tests/console

**Comment by danmosemsft** [comment]
```

Assert failure(PID 22 [0x00000016], Thread: 122 [0x007a]): Assertion failed '!foundDiff' in 'OpenSsl:GetSslError(Microsoft.Win32.SafeHandles.SafeSslHandle,int,byref):int' during 'Linear scan register alloc' (IL size 99)

    File: /__w/1/s/src/coreclr/src/jit/lsra.cpp Line: 2262
    Image: /root/helix/work/correlation/dotnet

./RunTests.sh: line 161:    22 Aborted                 "$RUNTIME_PATH/dotnet" exec --runtimeconfig System.Net.Requests.Tests.runtimeconfig.json --depsfile System.Net.Requests.Tests.deps.json xunit.console.dll System.Net.Requests.Tests.dll -xml testResults.xml -nologo -nocolor -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing $RSP_FILE
```

**Comment by danmosemsft** [comment]
@maryamariyan the bot should have added 'untriaged' ?

**Comment by AntonLapounov** [comment]
Hit it in my PR as well:
runtime (Libraries Test Run checked coreclr Linux_musl arm64 Release)
https://helix.dot.net/api/2019-06-17/jobs/c3ca6d1c-8d69-4a13-96ba-88e69e6cf1dd/workitems/System.Net.Requests.Tests/console
https://helix.dot.net/api/2019-06-17/jobs/c3ca6d1c-8d69-4a13-96ba-88e69e6cf1dd/workitems/System.Net.Security.Tests/console

**Comment by AndyAyersMS** [comment]
cc @CarolEidt 

Possibly a dup of #39221

**Comment by maryamariyan** [comment]
@danmosemsft I'll take a look

**Comment by BruceForstall** [comment]
Also hits in the `runtime-coreclr libraries-jitstress` job:

https://dev.azure.com/dnceng/public/_build/results?buildId=734948&view=ms.vss-test-web.build-test-results-tab&runId=22752754&resultId=173996&paneView=dotnet-dnceng.dnceng-build-release-tasks.helix-test-information-tab

**Comment by BruceForstall** [comment]
Another case from the libraries-jitstress job, for

net5.0-Linux-Release-arm64-CoreCLR_checked-zapdisable, System.Runtime.Extensions.Tests:
```
COMPlus_ReadyToRun=0
COMPlus_TieredCompilation=0
COMPlus_ZapDisable=1
+ ./RunTests.sh --runtime-path /root/helix/work/correlation
----- start Fri Jul 17 08:44:19 UTC 2020 =============== To repro directly: =====================================================
pushd .
/root/helix/work/correlation/dotnet exec --runtimeconfig System.Runtime.Extensions.Tests.runtimeconfig.json --depsfile System.Runtime.Extensions.Tests.deps.json xunit.console.dll System.Runtime.Extensions.Tests.dll -xml testResults.xml -nologo -nocolor -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing 
popd
===========================================================================================================
/root/helix/work/workitem /root/helix/work/workitem
  Discovering: System.Runtime.Extensions.Tests (method display = ClassAndMethod, method display options = None)
  Discovered:  System.Runtime.Extensions.Tests (found 825 of 882 test cases)
  Starting:    System.Runtime.Extensions.Tests (parallel test collections = on, max threads = 4)

Assert failure(PID 133 [0x00000085], Thread: 146 [0x0092]): Assertion failed '!foundDiff' in 'System.Environment:TryGetUserNameFromPasswd(long,int,byref):bool' during 'Linear scan register alloc' (IL size 112)

    File: /__w/1/s/src/coreclr/src/jit/lsra.cpp Line: 2262
    Image: /root/helix/work/correlation/dotnet
```

**Comment by BruceForstall** [comment]
Another, from

net5.0-Linux-Release-arm-CoreCLR_checked-jitstress2, System.IO.FileSystem.Tests:
```
COMPlus_TieredCompilation=0
COMPlus_JitStress=2
COMPlus_DbgMiniDumpName=/home/helixbot/dotnetbuild/dumps/coredump.%d.dmp
COMPlus_DbgEnableMiniDump=1
+ ./RunTests.sh --runtime-path /root/helix/work/correlation
----- start Fri Jul 17 08:07:40 UTC 2020 =============== To repro directly: =====================================================
pushd .
/root/helix/work/correlation/dotnet exec --runtimeconfig System.IO.FileSystem.Tests.runtimeconfig.json --depsfile System.IO.FileSystem.Tests.deps.json xunit.console.dll System.IO.FileSystem.Tests.dll -xml testResults.xml -nologo -nocolor -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing 
popd
===========================================================================================================
/root/helix/work/workitem /root/helix/work/workitem
  Discovering: System.IO.FileSystem.Tests (method display = ClassAndMethod, method display options = None)
  Discovered:  System.IO.FileSystem.Tests (found 2666 of 3311 test cases)
  Starting:    System.IO.FileSystem.Tests (parallel test collections = on, max threads = 4)
    System.IO.Tests.Directory_CreateDirectory.RootPath_AppContainer [SKIP]
      Condition(s) not met: "IsInAppContainer"

Assert failure(PID 26 [0x0000001a], Thread: 44 [0x002c]): Assertion failed '!foundDiff' in 'System.IO.Tests.FileStream_Dispose:FinalizeFlushesWriteBuffer():this' during 'Linear scan register alloc' (IL size 150)

    File: /__w/1/s/src/coreclr/src/jit/lsra.cpp Line: 2262
    Image: /root/helix/work/correlation/dotnet
```

**Comment by CarolEidt** [comment]
I believe that this was fixed with #39452.
